### ISSUE TITLE
How to change radio stream address ?

### ISSUE THREAD

**Issue by sebasonic** [segler-alex/radiobrowser#211]
Hi,
Is there a way to update radio station details ?
Also, is it possible to handle several streams on a radio for multiples qualities ?
thanks
Sebastien
https://chaos-radio.fr

**Comment by kBite** [comment]
I second that. Only after saving I noticed the URL I provided is not playable in browser.
> browser is not able to play station. please try with external player.

I found a browser-playable URL and would like to change it.

**Comment by sebasonic** [comment]
Hi here are our new streams URL
[https://chaos-radio.fr/stream/128](https://chaos-radio.fr/stream/128)
[https://chaos-radio.fr/stream/320](https://chaos-radio.fr/stream/320)


Thanks
Sébastien

**Comment by sidney65** [comment]
Hi , 

I have the same problem to update stream info of my station !
Someone have a solution ,address,website or someone else ?

**Comment by stemy2** [comment]
Same here. Some URLs are not valid anymore since long time but the radios still on the list. I need to update it, but it seems that i can't.

**Comment by sidney65** [comment]
Hi Alex,

For Metal Invasion Radio, the New stream url are :

https://radio.metal-invasion.fr/radio/8000/stream96


Please update it.

Do you need someone else?

Regards


Sidney

**Comment by olaf4** [comment]
Same issue here.
I want to change the stream urls of some German radio stations and also of some streaming stations, that have changed over time. No way to do that.
Please add an option for users to update/change a stream url. At least when it is clear, that the old url is broken.
Creating a new dataset with the valid stream is no option, because the broken one will remain in the data base. Also what if the stream url is changed again?

**Comment by kousu** [comment]
This is a duplicate of https://github.com/segler-alex/radiobrowser/issues/93, https://github.com/segler-alex/radiobrowser/issues/236 and https://github.com/segler-alex/radiobrowser/issues/118, and https://github.com/segler-alex/radiobrowser-api/issues/39. But now apparently the new tracking issue for this is: https://github.com/segler-alex/radiobrowser-api-rust/issues/49#issuecomment-578537820, in the backend's repo.

**Comment by kousu** [comment]
If you can contribute to that issue, please do. There's a lot of enthusiasm to get this fixed.

For now you have to add duplicate stations. At least, if you're using the android app, you can favourite the new one there. :shrug:

**Comment by segler-alex** [comment]
https://github.com/segler-alex/radiobrowser-api-rust/issues/49
